src/omniperf

```diff
@@ -56,15 +59,24 @@ def verify_deps_version(localVer, desiredVer, operator):
 
 
 def verify_deps():
-    """Utility to read library dependencies from requirements.txt and endeavor to load them within current execution environment. 
-    Used in top-level omniperf to provide error messages if necessary dependencies are not available. """
+    """Utility to read library dependencies from requirements.txt and endeavor
+    to load them within current execution environment.
+    Used in top-level omniperf to provide error messages if necessary
+    dependencies are not available."""
+
+    # Check which version of python is being used
+    if sys.version_info[0] < 3 or (sys.version_info[0] == 3 and sys.version_info[1] < 8):
+        print("[ERROR] Python 3.8 or higher is required to run omniperf."
+              f" The current version is {sys.version_info[0]}.{sys.version_info[1]}.")
+        sys.exit(1)
+
```

Comment:
Can we integrate Omniperf's built-in logging module so that we're capturing and recording this error in the same way we do elsewhere in Omniperf. i.e.

```python
from utils.utils import console_error

# Other code...

# Check which version of python is being used
    if sys.version_info[0] < 3 or (sys.version_info[0] == 3 and sys.version_info[1] < 8):
        console_error("Python 3.8 or higher is required to run omniperf."
              f" The current version is {sys.version_info[0]}.{sys.version_info[1]}.")
```

Reply:
Closer review of git blame reveals that generic handling using `print` and `sys.exit` are best suited for this function. Particularly, because this is where we handle dependency resolution; e.g. if the `logging` module is missing, we need the most generic form of error handling